Snowdog_Components/components/Organisms/contact-bar/_contact-bar-variables.scss

```diff
@@ -7,6 +7,7 @@ $contact-bar__display-from                       : $screen-l !default;
 $contact-bar__container-width                    : calc(100% - #{$spacer--medium}) !default;
 $contact-bar__container-direction                : row-reverse !default;
 $contact-bar__container-justify                  : space-between !default;
+$contact-bar__container-threshold                : $max-content-width + 2 * $spacer;
```

Comment:
To match width of https://github.com/SnowdogApps/magento2-alpaca-theme/blob/master/Snowdog_Components/components/Atoms/container/_container.scss#L18